### ISSUE TITLE
[Committer Swag 2022]  Work with the Eclipse Foundation and Shipping

### ISSUE THREAD

**Issue by emecas** [microprofile/microprofile-marketing#717]
### Overview

The purpose of this issue is following the work with Eclipse Foundation about Committer Swag 2022 and its shipping process

### Action Items

- [x] Open a discussion about to asset interest and people involved 
- [ ] Collect committers information for shipping (set up an spreadsheet)
- [ ] Swag's buying 
- [ ] Shipping process

### Resources/Instructions

- [x] Once #712 is complete , We will start the conversation about how to approach this issue
- [x] This is also linked to completion of #698 
- [x] #721
- [x] #720
- [x] #703

**Comment by aeiras** [comment]
linked to:

- [ ]  https://github.com/eclipse/microprofile-marketing/issues/721
- [ ] https://github.com/eclipse/microprofile-marketing/issues/720
- [ ] https://github.com/eclipse/microprofile-marketing/issues/712
- [ ] https://github.com/eclipse/microprofile-marketing/issues/703
- [ ] https://github.com/eclipse/microprofile-marketing/issues/698

**Comment by aeiras** [comment]
@emecas, 

I have added new tickets created to start this work as a checklist, you would add them under the Resources section of the ticket, great form. You rock!

**Comment by emecas** [comment]
> @emecas,
> 
> I have added new tickets created to start this work as a checklist, you would add them under the Resources section of the ticket, great form. You rock!

Done, Thx :) , #712 and #698  were already there, then I added to other 3

**Comment by aeiras** [comment]
Hola [emecas](https://github.com/emecas)

What is the status on this ticket, which relates to the other 2 tickets? Thank you for doing such a job moving all it forward.

**Comment by aeiras** [comment]
We need confirmation that it is completed to initiate purchase and the work for Committers and Contributors hoodies and shirts. ETA early Nov. The designs have been completed and approved by a few committers already. FYI on why my question.

**Comment by emecas** [comment]
### Last Activity (Oct)

- _Oct/10/2022_ : Burbank Shipping Center (BSC) picked up about 300+ shirts (5 boxes) from Deluxe (Elsie), and asked for an invoice of what is going where and finalized details on payment
- _Oct/10/2022_ : After Spoken with Monique from BSC, copied Eclipse Foundation (EF) People (3) to the email chain attempting to unblock the payment, informing them the possibility of providing a link for the payment offered by BSC
- _Oct/11/2022_ : Shanda replied to the email asking for additional context and clarifications about the request and the plan with these t-shirts.
- _Oct/13/2022_ : I replied to the email chain with all contexts about the request and a description about the action item expected from EF to complete the payment
- _Oct/14/2022_ : Shanda replied email and mentioned that she will connect with Monique to set up payment
- _Oct/27/2022_ : We got a request for the status on this ticket, then I personally replied to the email chain for a quick follow up
- _Oct/28/2022_ : BSC responded, confirming that EF got into contact with them directly with payment info, also that BSC have sent out all but 2 shipments, and asking for contact numbers for the 2 orders in pending because of international customs.
- _Oct/28/2022_ : I collected the phone numbers and provided them to BCS

**Comment by aeiras** [comment]
Just incredible work mediating this task, Emerson. You rock!!

BSC via email mentions  2 xs extras after distribution.  I am in LA, so I can get those items from BSC at a later time. Can you please unblock her on that question, @emecas ? 

Thank YOU! 
Next task is ThankYouSwag, which is awaiting until the learning from this task is delivered by beautiful you. :)

**Comment by emecas** [comment]
> BSC via email mentions 2 xs extras after distribution. I am in LA, so I can get those items from BSC at a later time. Can you please unblock her on that question, @emecas ?

DONE :)

**Comment by emecas** [comment]
> We need confirmation that it is completed to initiate purchase and the work for Committers and Contributors hoodies and shirts. ETA early Nov. The designs have been completed and approved by a few committers already. FYI on why my question.

### Last Activity (Nov) - Confirmation about [Swag 2021] Distribution logistics #688

- _Nov/01/2022_ : Burbank Shipping Center, attached the invoice that has all the names, addresses and tracking numbers for each recipient. Everything is shipped out.

**Comment by aeiras** [comment]
That's fantastic, @emecas!!!!

What was the cost of shipping costs?

**Comment by emecas** [comment]
> That's fantastic, @emecas!!!!
> 
> What was the cost of shipping costs?

US$ 1287.23

This is the invoice summary section:

![image](https://user-images.githubusercontent.com/849502/199360315-fa065f9c-0d6d-472f-8d61-90831a1e86a0.png)

**Comment by aeiras** [comment]
Emerson, 

Do you have the capability to adjust the invoice received to make it public view? 

The goal is to add it to the MP community folder under Swag tab> Orders-Quotes> 2021>Purchased
https://drive.google.com/drive/u/0/folders/17-U4DbUpnYzbiH9qkA6IJyMCK9XxZw36

Please see the 380 shirts order in the folder and the information that is publicly OK to be available. There shouldn't be any emails from us in the folder. That data and all personal data should be removed.

**Comment by aeiras** [comment]
> 

Also, there are 10 packages for delivery. Invoice says 8. 
Does that mean that 2 extra packages, the ones you found the missing information, still need to be charged? 

Thank you, beautiful you!

**Comment by emecas** [comment]
> Emerson,
> 
> Do you have the capability to adjust the invoice received to make it public view?
> 
I think so , give me some time to remove names, addresses, track numbers and  any other personal information, since it contains all details for each of the destination communities 

> The goal is to add it to the MP community folder under Swag tab> Orders-Quotes> 2021>Purchased https://drive.google.com/drive/u/0/folders/17-U4DbUpnYzbiH9qkA6IJyMCK9XxZw36
> 
> Please see the 380 shirts order in the folder and the information that is publicly OK to be available. There shouldn't be any emails from us in the folder. That data and all personal data should be removed.

**Comment by aeiras** [comment]
Excellent, Emerson. :) 

Nothing personal can be shared, not today, not ever in the MP public drives.  Please don't rush that work and do add yourself to this [ticket](https://github.com/eclipse/microprofile-marketing/issues/727) once you can close this ticket with that invoice being dropped to the drive and link add under Minutes for the next call. We will update the budget spreadsheet as well once we know all 10 packages' pricing. Currently, I see only 8 packages.

**Comment by emecas** [comment]
> > 
> 
> Also, there are 10 packages for delivery. Invoice says 8. Does that mean that 2 extra packages, the ones you found the missing information, still need to be charged?

yes , you are right , I calculated the total and the 2 last packages are still in pending to be added to the invoice, the final total is going to be US$ 1732.08, I already asked an updated invoice and then I will take care of anything else in pending to close this ticket!!! :) 

 


 
> Thank you, beautiful you!

**Comment by emecas** [comment]
These are the 2 invoices, ~I don't have access to the driver to drop them there~ 

UPDATE: now that I got access to upload documents to the folder, I already dropped them and included the links here:

![image](https://user-images.githubusercontent.com/849502/199378166-6af11ce2-6a99-43dc-b9af-84ec926c04dc.png)

[11_01_2022-BurbankShippingCenter-2-packages-Invoice158415.pdf](https://github.com/eclipse/microprofile-marketing/files/9915188/11_01_2022-BurbankShippingCenter-2-packages-Invoice158415.pdf)

https://drive.google.com/file/d/1hz6p-__msjLCbTsPdIT-Ow2ttoml1hMp/view?usp=share_link



![image](https://user-images.githubusercontent.com/849502/199378198-891d6409-5ff9-4ade-9d64-564c93182e8a.png)

[10_24_2022-BurbankShippingCenter-8-packages-Invoice158155.pdf](https://github.com/eclipse/microprofile-marketing/files/9915189/10_24_2022-BurbankShippingCenter-8-packages-Invoice158155.pdf)

https://drive.google.com/file/d/1m5v1KlfzDErIRe4S2qIS8Fo7Ihlx600U/view?usp=share_link

**Comment by aeiras** [comment]
@emecas, 

You are so inspiring, Emerson.  Enjoy your beyond deserved access to editing and maintaining the MP community drive with current and future work. 

[[Infra] MP Community drive access - Edit - to Emerson #732](https://github.com/eclipse/microprofile-marketing/issues/732)

**Comment by emecas** [comment]
We Agreed that main subject about this issue is completed during Marketing call 10/07/2022

The additional derived tasks related to [Committer Swag 2022] will be manage in individual issues:

Action Items to be created:

- Second round to collect committers information in pending for shipping (set up an spreadsheet)
- Swag's buying
- Shipping process

**Comment by aeiras** [comment]
@emecas, 

Today, we discussed the swag items to be purchased. Please check the agenda minutes. :)
